deisctl/config/config_test.go

```diff
@@ -2,11 +2,124 @@ package config
 
 import (
 	"encoding/base64"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"testing"
 )
 
+type store struct {
```

Comment:
hmm, `store` is a little misleading here, since it's just representing a single key/value pair. let's use `KVPair` instead, with a store being a slice of key/value pairs. For example:

```
type KVPair struct {
    Key string
    Value string
}
type MockStore []KVPair
```

That way it's more clear to the reader :)